### PR TITLE
WIP: Default family

### PR DESCRIPTION
**Pull request by alexeymorozov**
See fontist/fontist#247, #322

### CODE REVIEW

File: fonts/manifest.yml [hunk 1]
```diff
@@ -35,31 +35,47 @@ Cambria Math:
...
 STIX Two Math:
   - Regular
 
 Source Han Sans:
   - Bold
-  - Normal
+  - Regular
+
+Source Han Sans Normal:
   - Regular
```
Comment (alexeymorozov) [inline fonts/manifest.yml:79]: @Intelligent2013 Seems I need help with debugging. By some reason it throws this error, though a path for "Source Han Sans Normal" is returned from fontist.

```
Transforming...
java.io.IOException: Can't read /home/runner/.metanorma/fonts/SourceHanSans-Normal.ttc
	at java.desktop/java.awt.Font.checkFontFile(Font.java:1244)
	at java.desktop/java.awt.Font.createFont(Font.java:1224)
	at org.metanorma.fop.fontConfig.updateFontsForGraphicsEnvironment(fontConfig.java:688)
```

Comment (Intelligent2013) [reply fonts/manifest.yml:79]: @alexeymorozov I have to update mn2pdf for a new fontist format.
Before:
```
Work Sans:
  - Black
  - Black Italic
  - ...
```
New:
```
Work Sans Black:
  - Regular
  - Italic
 ```

Comment (Intelligent2013) [reply fonts/manifest.yml:79]: Just FYI, mn2pdf updated (v1.35b release for testing purposes only) for support both fontist formats (tested here https://github.com/metanorma/mn-native-pdf/pull/324).

Comment (alexeymorozov) [reply fonts/manifest.yml:79]: Thanks :) Let's rerun the build then

**Comment by alexeymorozov** [comment]
Merged in 2cf2d400a85234d8b8fef6a7aaa887ed81b6cb86

**Pull request closed without merging**